modules/ensemble/lib/framework/logging/log_provider.dart

```diff
@@ -8,8 +8,7 @@ abstract class LogProvider {
   bool shouldAwait = false;
   Map? options;
   String? ensembleAppId;
-  Future<void> log(
-      String event, Map<String, dynamic> parameters, LogLevel level);
+  Future<void> handleAnalytics(Map<String, dynamic> config);
```

Comment:
keep it as log please. You are just logging things, it's not handling analytics.